src/components/RenderInsideChart.jsx

```diff
@@ -25,7 +25,12 @@ class RenderInsideChart extends Component {
         });
     }
 
+    componentDidMount() {
+        reaction(() => this.props.isChartReady, () => this.forceUpdate());
```

Comment:
@nazanin-binary Does it need to use `reaction` and `forceUpdate` while `isChartReady` is observable and part of the component property?

Reply:
right it's not needed, will remove it, thanks